x-pack/plugins/security_solution/package.json

```diff
@@ -25,7 +25,8 @@
     "mappings:generate": "node scripts/mappings/mappings_generator",
     "mappings:load": "node scripts/mappings/mappings_loader",
     "junit:transform": "node scripts/junit_transformer --pathPattern '../../../target/kibana-security-solution/cypress/results/*.xml' --rootDirectory ../../../ --reportName 'Security Solution Cypress' --writeInPlace",
+    "openapi:lint": "npx @redocly/cli lint --config ../../../packages/kbn-openapi-generator/redocly_linter/config.yaml $(find common/api -name '*.schema.yaml')",
```

Comment:
When running this script locally for the first time, `npx` interactively asks if I want to install the `@redocly/cli` package:

```sh
    yarn openapi:lint
yarn run v1.22.19
$ npx @redocly/cli lint --config ../../../packages/kbn-openapi-generator/redocly_linter/config.yaml $(find common/api -name '*.schema.yaml')
Need to install the following packages:
@redocly/cli@1.4.1
Ok to proceed? (y) y
validating common/api/endpoint/model/schema/common.schema.yaml...
```

How does it work in CI where we don't have interactivity?

Wouldn't it be better to add `@redocly/cli` to dev dependencies?

Reply:
I've installed `@redocly/cli` in [this](https://github.com/elastic/kibana/pull/171851/commits/e5c6464353a7b7fa4038ac1153e4b0482004e7e4) commit.